years.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if (grep -r "Copyright \+(c) \+2009-.*rultor.com" \
+  --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . \
+  | grep -v 2009-`date +%Y`); then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+fi
+
+if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
+ --include=*.java --include=*.xml --include=*.vm --include=*.groovy \
```

Comment:
@vkuchyn minor: indentation here is one space, while above for ` --exclude-dir` you have two spaces